subsys/bluetooth/host/conn.c

```diff
@@ -428,7 +428,15 @@ int bt_conn_send_cb(struct bt_conn *conn, struct net_buf *buf,
 	if (cb) {
 		tx = conn_tx_alloc();
 		if (!tx) {
-			LOG_ERR("Unable to allocate TX context");
+#if defined(CONFIG_BT_CONN)
+			if (!(net_buf_pool_get(buf->pool_id) == &acl_tx_pool)) {
+#else
+			if (false) {
+#endif
```

Comment:
Seems like the above comment was left in my cache (as you can see from the timestamp). Not sure if it's relevant or correct anymore, since the PR has probably evolved since then.